### PR TITLE
apps/system/utils: Fix memory leak issue in ping command.

### PR DESCRIPTION
**Pull request by xiarihf**
1. Release the heap when ping_recv function returns.
    2. Fix coding rule issue.

    Signed-off-by: xiarihf qiang3.zhang@samsung.com

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [544206ffe03716ce5471394642bd82874d47a97d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [544206ffe03716ce5471394642bd82874d47a97d] - Code Rule Check Result:
apps/system/utils/netcmd_ping.c:73: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
apps/system/utils/netcmd_ping.c:248: ERROR: [IDT_M_SCH] switch and case should be at the same indent
apps/system/utils/netcmd_ping.c:468: ERROR: [IDT_M_TAB] please, no spaces at the start of a line

**Comment by seinfra** [comment]
Target : [92206790147f977cc20672d9a3df44a9a411ac0b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [92206790147f977cc20672d9a3df44a9a411ac0b] - Code Rule Check OK.

File: apps/system/utils/netcmd_ping.c [hunk 1]
```diff
@@ -70,7 +70,7 @@
 #define ICMP_HDR_SIZE sizeof(struct ip_hdr) + sizeof(struct icmp_echo_hdr)
 #ifdef CONFIG_NET_IPv6
 #define ICMP6_HDR_SIZE sizeof(struct ip6_hdr) + sizeof(struct icmp6_echo_hdr)
-#define PING_SIZE(proto) ((proto == IPPROTO_ICMPV6) ? ICMP6_HDR_SIZE : ICMP_HDR_SIZE )
+#define PING_SIZE(proto) ((proto == IPPROTO_ICMPV6) ? ICMP6_HDR_SIZE : ICMP_HDR_SIZE)
```
[the remark this answers is not in the document]
Comment (xiarihf) [reply apps/system/utils/netcmd_ping.c]: fixed, thanks.

**Comment by seinfra** [comment]
Target : [b79ea3f4515d2fdcedb296299c081e8cfd8ea89f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b79ea3f4515d2fdcedb296299c081e8cfd8ea89f] - Code Rule Check OK.

**Comment by xiarihf** [comment]
@sunghan-chang 
Hi, Mr. chang,  can you help to review this commit?  
Please help to merge it, if it's OK, Thanks.

**Comment by sunghan-chang** [comment]
@xiarihf Hello. I am sorry but I am not working on network feature.
@pillip8282 Could you review this?

**Pull request merged**